tests/test_utils.h

```diff
@@ -77,34 +77,34 @@ std::string TEST_EXPORT resourcePath(const std::string &resourceRelativePath = "
 
 std::string TEST_EXPORT fileContents(const std::string &fileName);
 
-void TEST_EXPORT printErrors(const libcellml::LoggerPtr &l, bool headings = false, bool kinds = false, bool rule = false);
+void TEST_EXPORT printIssues(const libcellml::LoggerPtr &l, bool headings = false, bool causes = false, bool rule = false);
 
 void TEST_EXPORT printComponent(const libcellml::ComponentPtr &component, size_t const c, std::string const spacer);
 
 void TEST_EXPORT printModel(const libcellml::ModelPtr &model);
 
-void TEST_EXPORT expectEqualErrors(const std::vector<std::string> &errors,
-                                   const libcellml::LoggerPtr &logger);
-void TEST_EXPORT expectEqualErrorsSpecificationHeadings(const std::vector<std::string> &errors,
+void TEST_EXPORT expectEqualIssues(const std::vector<std::string> &issues, const libcellml::LoggerPtr &logger);
+
```

Comment:
Remove empty line (i.e. as it was before)?

Reply:
Done